files/galaxy/tpv/tool_defaults.yml

```
@@ -29,6 +29,17 @@ tools:
           entity.params['requirements'] = '(GalaxyGroup == "compute") || (%s)' % training_expr if training_expr else '(GalaxyGroup == "compute")'
           entity.params['+Group'] = training_labels
           entity.params['accounting_group_user'] = str(user.id)
+      - id: remote_resources
+        if: user is not None
+        execute: |
+          user_preferences = user.extra_preferences
+          pulsar_tag = None
+          for data_item in user_preferences:
+            if "distributed_compute|remote_resources" in data_item:
+                if user_preferences[data_item] != "None":
+                    pulsar_tag = user_preferences[data_item]
+          new_tag = [{ 'require': pulsar_tag }] if pulsar_tag else []
```

```suggestion
          new_tag = { 'require': pulsar_tag } if pulsar_tag else {}
```

_See [this comment](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/775/files#r1211557549)._